### ISSUE TITLE
请教一下 

### ISSUE THREAD

**Issue by yinweidong1992** [yellowtownhz/STIGCN#4]
# cheby
        A3 = A[1] + A[2]
        self.A3 = 4*torch.pow(A3, 2) - A3 - 2*torch.eye(A3.size(-1))
        self.PA3 = nn.Parameter(self.A3.clone(), requires_grad=True)
        nn.init.constant_(self.PA3, 1e-6)

        soft = nn.Softmax(-2)
        self.A4 = soft((8*torch.pow(A3, 4)- 4*torch.pow(A3, 2)-4*A3 +torch.eye(A3.size(-1)))/A3.size(-1))
        self.PA4 = nn.Parameter(self.A4.clone(), requires_grad=True)

这个里面， self.A3   self.A4  为何这样算，怎么转换到论文中的T3 ， T4
